Dockerfile.ci

```diff
@@ -1,8 +1,5 @@
-ARG BUILDER_IMAGE=registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19
-ARG BASE_IMAGE=registry.ci.openshift.org/ocp/builder:rhel-9-base-openshift-4.19
```

Comment:
This usage has no issue. It just should be `registry.ci.openshift.org/ocp/4.19:base-rhel9` like https://github.com/openshift/kubernetes-sigs-lws/blob/55e64b90af76b7e76c6476b7aa8fc22698833ab0/Dockerfile.ci#L2C16-L2C61

Builder one ^^ is correct.